pkg/cloud/azure/actuators/machine/actuator_test.go

```diff
@@ -203,8 +205,8 @@ func (s *FakeVMService) Get(ctx context.Context, spec azure.Spec) (interface{},
 	}, nil
 }
 
-// CreateOrUpdate returns fake success.
-func (s *FakeVMService) CreateOrUpdate(ctx context.Context, spec azure.Spec) error {
+// Reconcile returns fake success.
+func (s *FakeVMService) Reconcile(ctx context.Context, spec azure.Spec) error {
```

Comment:
Please add test cases

1. Get is called
2. CreateOrUpdate is called only when expected != current 

Rename the fields in test cases here